### DISCUSSION TITLE
A mini guide or some tips on how to use it with Dokku?

### DISCUSSION THREAD

**Discussion by Richi2293** [mbround18/valheim-docker#69] [Instructions]
Hi,

It would be interesting to be able to use it with Dokku, I've been trying for days but still without success.
I almost certainly have a problem with the ports because the server starts fine.

Thanks

**Comment by mbround18** [comment]
I had no idea Dokku existed! :P I suspect if Dokku is a clone of heroku you might have issues running anything that utilizes UDP. Although I might be wrong, I can spin up a Dokku instance and give it a whirl in a few days :)

**Reply by Richi2293** [reply]
Ah okok, thanks :)

**Comment by TkTech** [comment]
I wrote down a small guide here, as notes for future me: https://tkte.ch/articles/2023/03/03/Valheim.html

**Reply by mbround18** [reply]
@TkTech added to readme in my pr #669! :)
https://github.com/mbround18/valheim-docker/pull/669/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R56
